ios/Podfile.lock

```diff
@@ -1,4 +1,7 @@
 PODS:
+  - connectivity (0.0.1):
```

Comment:
Can we rather ignore Podfile.lock and remove from repository? In my understanding, it will be generated automatically during iOS build as github action, right?

Reply:
yes should be removed.